elcid/templates/inline_forms/clinical_timeline.html

```diff
@@ -72,6 +72,6 @@
 {% endblock %}
 
 {% block send_upstream_button %}
-<i class="fa fa-exchange edit pull-right pointer"
+<i ng-show="item.reason_for_interaction && item.reason_for_interaction.indexOf('RNOH') !== 0" class="fa fa-exchange edit pull-right pointer"
```

Comment:
Wouldn't a `-1` spelling of this be safer against future spellings of reason for interaction?

Reply:
I thought I was future proofing by making sure we didn't catch false positives. 

I was wary we might have a RNOH in the middle of a non RNOH reason for interaction (e.g. discharge not for patient going to RNOH). I'm happy either way.